### PR TITLE
i18n(fr): Translate unsupported-config-transform-error.mdx

### PR DESCRIPTION
**Pull request by MartinFerret**
This file, french version

### CODE REVIEW

File: src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+titre : Transformation non prise en charge dans la configuration du contenu. 
+i18nPrêt : true 
+githubURL : https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
```
Comment (shuuji3) [inline src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx:4]:
```suggestion
title: Transformation non prise en charge dans la configuration du contenu. 
i18nReady: true 
githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
```

**Review by shuuji3 — CHANGES_REQUESTED** [review]
Could you try this change?

File: src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx (hunk 1 above)
Comment (sarah11918) [inline src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx:4]:
```suggestion
title: Transformation non prise en charge dans la configuration du contenu. 
i18nReady: true 
githubURL : https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
```
Frontmatter values don't get translated! This should fix the errors!  🙌

File: src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Transformation non prise en charge dans la configuration du contenu. 
+i18nReady: true 
+githubURL : https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
```
Comment (sarah11918) [inline src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx:4]:
```suggestion
title: Transformation non prise en charge dans la configuration du contenu. 
i18nReady: true 
githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
```

File: src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx (hunk 1 above)
Comment (sarah11918) [reply src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx]: Good call! I was fixing these at the same time! 😄

**Comment by shuuji3** [comment]
@sarah11918 Thank you for the update! 🙂

File: src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Transformation non prise en charge dans la configuration du contenu. 
+i18nReady: true 
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+--- 
+
+import DontEditWarning from '~/components/DontEditWarning.astro'
```
Comment (VoxelMC) [inline src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx:7]:
```suggestion
```

I believe you can safely remove this import, as the page is being translated and so doesn't need the component anymore.

**Review by VoxelMC — COMMENTED** [review]
Left one comment. Otherwise, it looks great! Thank you for your hard work. 

Once the line and whitespace is removed, I'd say: `As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

**Review by McFlyPartages — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps, and welcome to Team Docs @MartinFerret!

**Pull request merged**